src/fdk_organization_bff/classes/catalog_quality_score.py

```diff
@@ -4,8 +4,8 @@
 
 
 @dataclass
-class CatalogQualityRating:
-    """Data class with quality assessment category and associated percentage."""
+class CatalogQualityScore:
+    """Data class with quality assessment score and associated percentage."""
 
-    category: Optional[str]
+    score: Optional[int]
```

Comment:
Bli eneste oppdateringen som trengs i frontend at smileyene skal bestemmes ut fra percentage i stedet for category?

Reply:
Ja og at percentage vises. Det er allerede gjort.